migrations-1/marketplace1.sql

```diff
@@ -0,0 +1,192 @@
+
+-- Run marketplace2.sql in marketplace2 container first , then run this file
+-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER node1 FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host 'marketplace2', dbname 'ds_db');
+CREATE USER MAPPING FOR postgres SERVER node1 OPTIONS (user 'postgres' , password '1700455');
```

Comment:
I mean we can't have the password lying here like this. I think we can use a templating engine and a placeholder value.